tests/unit/multi-currency/compatibility/test-class-woocommerce-product-add-ons.php

```diff
@@ -119,12 +119,12 @@ public function test_get_addons_price_returns_percentage_without_conversion() {
 	}
 
 	public function test_get_addons_price_returns_converted_price() {
-		$this->mock_multi_currency->method( 'get_price' )->with( 50, 'product' )->willReturn( 75 );
-		$this->assertEquals( 75, $this->woocommerce_product_add_ons->get_addons_price( 50, [ 'price_type' => 'flat_fee' ] ) );
+		$this->mock_multi_currency->method( 'get_price' )->with( 50.00, 'product' )->willReturn( 75.00 );
```

Comment:
Have we made any changes that lead to the precision of numbers here? We should ideally get the tests to work with the older values themselves for a refactor IMO. Might be wrong here in case this is a new quirk with the version bump itself so please CMIIAW.

Reply:
You are right. I will fix it.

Reply:
Fixed.